app/helpers/template_helpers.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self._include_csrf_token = include_csrf_token
         self._survey_config = survey_config
         self._survey_title = cookie_session.get(
-            "title", self._survey_config.survey_title
+            "survey_title", lazy_gettext("ONS Surveys")
```

Comment:
The title should only use the default when we have no cookie, "survey_title" shouldn't be overridden, this should still be "title". At the moment, all titles are suffixed with "ONS Surveys", but that should only be the case when we don't have a cookie.

Reply:
Ahh I think me changing `tests` to `survey_title` is why I have so many errors with the tests. The tests are working as they should now, just need to revert changes 👍

Reply:
Changes Reverted. Once this is resolved would it be okay to merge, since the only change made after the two approvals I previously got is changing `survey_title` to `title`.